### ISSUE TITLE
Spec is null error in web console

### ISSUE THREAD

**Issue by ghost** [iheartradio/play-swagger#306]
Hi there,

When i try to reach my swagger.json in swagger ui, I get this message: "fetching resource list: http://localhost:9000/assets/swagger.json; Please wait.". In the web console, an error is shown as:
```
TypeError: spec is null4 operation.js:1100
    resolve operation.js:1100
    response operation.js:249
    finish cookiejar.js:50
    convert index.js:1209
    response operation.js:247
    responseInterceptor operation.js:777
    response operation.js:792
    execute operation.js:1032
    callback swagger-ui.js:20615
    Request swagger-ui.js:20330
    emit swagger-ui.js:21240
    onreadystatechange swagger-ui.js:20697
```
Yet, the reponse body shows my API documentation. I'm trying to upgrade to openApi v3,
Thanks

**Comment by pmckelvy1** [comment]
I too am experiencing this issue. I can see the valid json for the swagger.json file in network tab in chrome, but the ui won't display it.

**Comment by lecointef** [comment]
@kailuowang maybe you can help on this ?

**Comment by kailuowang** [comment]
might be some configuration error. 
Try validate the json using swagger editor
https://editor.swagger.io/

**Comment by robertofabrizi** [comment]
Same error here, only seems to happen with Swagger 3.0. The output of the call to swagger.json is correct, I validated it with the editor:

```
components:
  schemas: {}
openapi: 3.0.0
paths:
  '/{instance}/console/api/{protocol}/cpu':
    get:
      summary: create a card
      responses:
        '200':
          description: success
      operationId: getProcessCpu
      parameters:
        - schema:
            type: string
          in: path
          name: instance
          required: true
        - schema:
            type: string
          in: query
          name: name
          required: true
        - schema:
            type: string
          in: query
          name: ip
          required: true
        - schema:
            format: int32
            type: integer
          in: query
          name: port
          required: true
        - schema:
            type: string
          in: path
          name: protocol
          required: true
      tags:
        - Card Endpoints
info:
  description: REST API
  title: API
  version: 1.0.0
tags:
  - name: routes
```

**Comment by robertofabrizi** [comment]
It seems fixed by upgrading the 2.2.0 ancient swagger-ui import with a more recent one: 

libraryDependencies += "org.webjars" % "swagger-ui" % "3.24.3"
